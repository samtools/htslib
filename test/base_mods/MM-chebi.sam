@CO	Separate m, h and N modifications
*	0	*	0	0	*	*	0	0	AGCTCTCCAGAGTCGNACGCCATYCGCGCGCCACCA	*	Mm:Z:C+m,2,2,1,4,1;C+76792,6,7;N+n,15;	Ml:B:C,102,128,153,179,204,161,33,212
