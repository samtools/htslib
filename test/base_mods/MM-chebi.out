0	A
1	G
2	C
3	T
4	C
5	T
6	C	C+m102
7	C
8	A
9	G
10	A
11	G
12	T
13	C
14	G
15	N	N+n212
16	A
17	C	C+m128
18	G
19	C	C+(76792)161
20	C	C+m153
21	A
22	T
23	Y
24	C
25	G
26	C
27	G
28	C
29	G
30	C
31	C	C+m179
32	A
33	C
34	C	C+m204 C+(76792)33
35	A
---
Present: m. #-76792. n.
6	C	C+m102
15	N	N+n212
17	C	C+m128
19	C	C+(76792)161
20	C	C+m153
31	C	C+m179
34	C	C+m204 C+(76792)33

===

